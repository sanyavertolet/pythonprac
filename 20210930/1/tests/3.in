91,85,79,44,39,78,23
