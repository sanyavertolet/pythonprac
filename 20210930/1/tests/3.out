[39, 85, 23, 44, 79, 91, 78]
